File: src/ru/serjik/preferences/controllers/SeparatorController.java
package ru.serjik.preferences.controllers;

import android.view.View;
import android.view.ViewGroup;
import ru.serjik.preferences.PreferenceController;

public class SeparatorController extends PreferenceController {
    @Override
    protected View createView(String[] params) {
        View view = new View(this.context);
        view.setLayoutParams(new ViewGroup.LayoutParams(-1, dp(8)));
        return view;
    }
}

File: app/src/main/java/ru/serjik/preferences/controllers/SeparatorController.kt
package ru.serjik.preferences.controllers

import android.view.View
import android.view.ViewGroup
import ru.serjik.preferences.PreferenceController

class SeparatorController : PreferenceController() {
    override fun createView(params: Array<String>): View {
        return View(context).apply {
            layoutParams = ViewGroup.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, dp(8))
        }
    }
}
